packages/semantic-tokens/src/control/tokens.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Control level tokens are generic tokens that can be used across multiple components.
+ *
+ * Components with a different fluent 2 fallback than the control should define a component specific token instead.
+ */
+// eslint-disable-next-line no-restricted-imports
+import { tokens } from '@fluentui/tokens';
+
+// Text
+export const textStyleDefaultRegularFontfamily = `var(--text-style-default-regular-fontfamily, ${tokens.fontFamilyBase})`;
+export const textGlobalBody3Fontsize = `var(--text-global-body3-fontsize, ${tokens.fontSizeBase300})`;
```

Comment:
Will need a system for layering in optional tokens.

Reply:
My understanding is that these tokens do not have an optional token layer, based on the JSON I recieved - we can add multiple layers if needed, but not sure we want multiple layers of optional tokens if not needed (for optimized bundle size) and some of these tokens are provided as component level tokens which will have an additional component-specific targeted CSSVar.

Consuming users can override '--text-style-default-regular-fontfamily' if looking to replace any preconfigured CSS vars (but in Fluent 2 this will be null and resolve to existing tokens.fontFamilyBase).